.circleci/config.yml

```diff
@@ -1,42 +1,48 @@
 ---
-default_job: &default_job
-  working_directory: ~/administrate
-  steps:
-    - checkout
+version: 2.1
 
-    # Restore Cached Dependencies
-    - type: cache-restore
-      name: Restore bundle cache
-      key: administrate-{{ checksum "Gemfile.lock" }}
+commands:
+  shared_steps:
+    steps:
+      - checkout
 
-    # Bundle install dependencies
-    - run: bundle install --path vendor/bundle
+      # Restore Cached Dependencies
+      - restore_cache:
+          name: Restore bundle cache
+          key: administrate-{{ checksum "Gemfile.lock" }}
 
-    # Install Appraisal gemfiles
-    - run: bundle exec appraisal install
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+
+      # Cache Dependencies
+      - save_cache:
+          name: Store bundle cache
+          key: administrate-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
 
-    # Cache Dependencies
-    - type: cache-save
-      name: Store bundle cache
-      key: administrate-{{ checksum "Gemfile.lock" }}
-      paths:
-        - vendor/bundle
+      # Wait for DB
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
 
-    # Wait for DB
-    - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      # Setup the environment
+      - run: cp .sample.env .env
 
-    # Setup the environment
-    - run: cp .sample.env .env
+      # Setup the database
+      - run: bundle exec rake db:setup
 
-    # Setup the database
-    - run: bundle exec rake db:setup
+      # Run the tests
+      - run: bundle exec rake
 
-    # Run the tests
-    - run: bundle exec rake
+default_job: &default_job
+  working_directory: ~/administrate
+  steps:
+    - shared_steps
+    # Run the tests against multiple versions of Rails
+    - run: bundle exec appraisal install
     - run: bundle exec appraisal rake
 
 jobs:
-  ruby-2.4:
+  ruby-24:
```

Comment:
Does Circle CI 2.1 require the removal of the dot here?

Reply:
Yeah it does. I had mentioned that in the commit, but forgot to add it to the PR description.

Reply:
Exciting.